plugins/node/opentelemetry-instrumentation-ioredis/src/utils.ts

```diff
@@ -94,7 +94,10 @@ export const traceSendCommand = (
       return original.apply(this, arguments);
     }
     // Do not trace if there is not parent span
-    if (getSpan(context.active()) === undefined) {
+    if (
+      config?.requireParentSpan !== false &&
```

Comment:
Because in description it says that it defaults to `true`
I think it should be

```suggestion
      config?.requireParentSpan === true &&
```

Then in ioredis.ts
```
const DEFAULT_CONFIG: IORedisInstrumentationConfig = {
  requireParentSpan: true,
}
```
and in ioredis contructor
```
  constructor(protected _config: IORedisInstrumentationConfig = {}) {
    super('@opentelemetry/instrumentation-ioredis', VERSION, Object.assign({}, DEFAULT_CONFIG, _config));
  }
```
WDYT ?

Reply:
Done :)